src/plugins/fs/common.c

```diff
@@ -192,7 +192,7 @@ wipe_fs (const gchar *device, const gchar *fs_type, gboolean wipe_all, GError **
     blkid_reset_probe (probe);
 
     if (wipe_all) {
-        has_fs_type = has_fs (probe, device, fs_type, error);
+        has_fs_type = has_fs (probe, device, NULL, error);
```

Comment:
The name of the variable no longer makes sense. And maybe the whole code can be simplified?

Reply:
Good point, fixed.